src/platform/forms-system/src/js/components/FormSignature.jsx

```diff
@@ -137,37 +141,35 @@ export const FormSignature = ({
 
 FormSignature.propTypes = {
   formData: PropTypes.object.isRequired,
+  // eslint-disable-next-line react/sort-prop-types
   onSectionComplete: PropTypes.func.isRequired,
```

Comment:
Disabled here because the sort rules conflicted - it was supposed to be at the top of the list AND the bottom simultaneously(?)